apps/emqx_gateway/etc/emqx_gateway.conf

```diff
@@ -4,167 +4,169 @@
 
 gateway: {
 
-    stomp.1: {
-        frame: {
-            max_headers: 10
-            max_headers_length: 1024
-            max_body_length: 8192
-        }
+  stomp: {
```

Comment:
We could reduce the indentations by changing the config to `gateway.stomp`, `gateway.coap`, ...